rules/do-you-know-how-to-delete-orphaned-users-from-the-people-picker/rule.md

```diff
@@ -11,20 +11,19 @@ redirects: []
 
 ---
 
-The SharePoint "People Picker" is a great tool for quickly selecting users of your SharePoint site. It allows you to quickly select users from your organization by browsing Active Directory. 
+The SharePoint "People Picker" is a great tool for quickly selecting users of your SharePoint site. It allows you to quickly select users from your organization by browsing Active Directory.
 <!--endintro-->
 ![](PeoplePicker.jpg)
```

Comment:
Hi @tombui99 

Markdown - When you are fixing links on a rule, it's a good idea to look for other issues

- [ ] 1. Please fix the Markdown for all images/captions in this rule

Reply:
Hey @tiagov8 

> Please fix the Markdown for all images/captions in this rule

Done - images fixed